config/locales/wicked.en.yml

```diff
@@ -0,0 +1,7 @@
+en:
```

Comment:
These i18n translations allow us to correspond `GET /steps/1` with being at `stage_1`, for instance.